auth/auth-model.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../database/dbConfig")
+
+module.exports = {
+  add,
+  findById,
+  findBy,
+};
+
+function findBy (filter) {
+  return db("users")
+    .where(filter)
+}
+
+async function add (user) {
+  try {
+    const [id] = await db("users").insert(user, "id")
+    return findById(id);
+  } catch (error) {
+    throw error;
```

Comment:
This try/catch isn't actually doing anything. It catches the error, then re-throws the exact same error. Removing it would change nothing, since the same error bubbles up either way. This would be a good place for something like `throw new Error('Cannot add user ${user.name}.')`.